website/docs/cloud-docs/workspaces/variables/managing-variables.mdx

```diff
@@ -148,6 +148,13 @@ To overwrite a variable from a variable set, [create a new workspace-specific va
 
 Variables within a variable set can also automatically overwrite variables with the same key in other variable sets applied to the same workspace. Though variable sets are created for the organization, these overwrites occur within each workspace. Refer to [variable precedence](/terraform/cloud-docs/workspaces/variables#precedence) for more details.
 
+## Priority Variable Sets
+
+The values in priority variable sets overwrite any variables with the same key set at more specific scopes. This includes variables set using command line flags, or through`.*auto.tfvars` and `terraform.tfvars` files.
+
+The only variable values that a prioritized variable set cannot overwrite are variables with values set directly in the terraform configuration itself. If you want to fully ensure the values of variables including values in your terraform configuration code, we recommend using a Policy Check or Run Task.
```

Comment:
```suggestion
While a priority variable set can enforce specific variable values, it does not guarantee that the configuration actually references that variable instead of hardcoding a value. For stricter enforcement mechanisms, we recommend using policy checks or run tasks.
```